ghost-gcs-backup/templates/cronjob.yaml

```diff
@@ -0,0 +1,62 @@
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: {{ template "ghost-gcs-backup.fullname" . }}
+  labels:
+    app: {{ template "ghost-gcs-backup.name" . }}
+    chart: {{ template "ghost-gcs-backup.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+spec:
+  concurrencyPolicy: Forbid
+  failedJobsHistoryLimit: 2
+  jobTemplate:
+    spec:
+      activeDeadlineSeconds: {{ default 60 .Values.job.activeDeadlineSeconds }}
+      backoffLimit: {{ default 5 .Values.job.backoffLimit }}
+      template:
+        spec:
+          restartPolicy: OnFailure
+          containers:
+          - name: {{ .Chart.Name }}
+            image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
+            imagePullPolicy: {{ .Values.image.pullPolicy }}
+            env:
+            {{- if .Values.googleApplicationCredentialsSecret.enabled }}
+            - name: GOOGLE_APPLICATION_CREDENTIALS
+              value: "/var/secrets/google/{{ required "googleApplicationCredentialsSecret.key" .Values.googleApplicationCredentialsSecret.key }}"
+            {{- end }}
+            - name: GHOST_ENDPOINT
+              value: "{{ required "Missing app.ghostEndpoint" .Values.app.ghostEndpoint }}"
+            - name: GHOST_API_KEY
+              value: "{{ required "Missing app.ghostAPIKey" .Values.app.ghostAPIKey }}"
```

Comment:
I don't think it is a good idea to place api key directly here, it's always better to use secrets. The approach I like the most is:

* If a secret is given, use it (preferred way)
* If a key is given in the values, create a secret and use it

See an example of this in https://github.com/src-d/code-annotation/tree/master/helm-charts/code-annotation